src/views/UrbitOS/AccessCode.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import AccessCode from './AccessCode';
+
+describe('AccessCode', () => {
+  describe('when code is not available', () => {
+    // Value is initialized as `false` in `useKeyfileGenerator`
+    const testCode = false;
+
+    it('does not render the component', () => {
+      render(<AccessCode code={testCode} />);
+      const accessCode = screen.queryByText(
+        'This is your code to access Urbit OS'
```

Comment:
Wouldn't it be better to just test the rendered document for being empty or not? The way the tests are currently written makes them depend on (and thus change with) the exact copy used, which I imagine we want to avoid? (I don't actually know much about interface testing, so you tell me!)

Reply:
>Wouldn't it be better to just test the rendered document for being empty or not?

Found a way to do so thanks to [the docs](https://testing-library.com/docs/guide-disappearance/#asserting-elements-are-not-present), good suggestion.

>The way the tests are currently written makes them depend on (and thus change with) the exact copy used, which I imagine we want to avoid?

It could be a good smoke test in the truthy case, it's a trivial way of ensuring we don't insert a typo in the copy while working on that file. Also, [it's recommended](https://testing-library.com/docs/guiding-principles) by `testing-library` to use text strings in tests, because it's simulates how users navigate the real app.